src/services/dashboards/modules/dashboard-toolset/DashboardToolset.vue

```diff
@@ -1,58 +1,50 @@
 <template>
     <div class="dashboard-toolset">
-        <dashboard-date-range-badge :date-range="state.proxyDateRange" />
-        <dashboard-date-dropdown :date-range.sync="state.proxyDateRange" />
-        <currency-select-dropdown default-mode />
+        <dashboard-date-range-badge v-show="props.enableDateRange"
+                                    :date-range="state.proxyDateRange"
+        />
+        <dashboard-date-dropdown v-show="props.enableDateRange"
+                                 :date-range.sync="state.proxyDateRange"
+        />
+        <currency-select-dropdown v-show="props.enableCurrency"
+                                  :default-currency="DEFAULT_CURRENCY"
```

Comment:
suggestion: if `defaultCurrency` is fixed value, it doesn't seem necessary to make it a props.

Reply:
`defaultCurrency` could be change, but it looks so far.
I'll change it! Thanks for your opinion